src/librbd/cache/WriteLogImageDispatch.cc

```diff
@@ -94,10 +81,10 @@ bool WriteLogImageDispatch<I>::write(
     return true;
   }
 
-  start_in_flight_io(aio_comp);
+  m_plugin_api.start_in_flight_io(aio_comp);
 
-  aio_comp->set_request_count(1);
-  io::C_AioRequest *req_comp = new io::C_AioRequest(aio_comp);
+  m_plugin_api.update_aio_comp(aio_comp, 1);
```

Comment:
Nit: you can probably combine the start w/ this method

Reply:
Updated